tests/unit/test_tasks.py

```
@@ -64,6 +64,98 @@ def test_cmd_converted_schema(self):
+
+	def test_truncated_xml_no_host_does_not_error(self):
+		"""XML truncated mid-scan (no completed host) is repaired and parses without errors."""
+		truncated = (
+			'<?xml version="1.0" encoding="UTF-8"?>\n'
+			'<!DOCTYPE nmaprun>\n'
+			'<nmaprun scanner="nmap" args="nmap -p - secator.cloud" start="1781863253" version="7.98">\n'
+			'<scaninfo type="connect" protocol="tcp" numservices="65535" services="1-65535"/>\n'
+			'<verbose level="0"/>\n'
+			'<taskprogress task="Connect Scan" time="1781863273" percent="12.10" remaining="139" etc="1781863411"/>\n'
+		)
+		errors, ports = self._parse(truncated)
```

Ruff (0.15.17)

[warning] Unpacked variable `ports` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)